mkdocs.yml

```diff
@@ -102,11 +102,12 @@ nav:
 
     - "Release notes" :
         - "Overview"                                                             : openj9_releases.md
+        - "Version 0.29.1"                                                       : version0.29.1.md
         - "Version 0.29.0"                                                       : version0.29.md
         - "Version 0.27.0"                                                       : version0.27.md
         - "Version 0.26.0"                                                       : version0.26.md
-        - "Version 0.25.0"                                                       : version0.25.md
         - "Earlier releases" :
+	        - "Version 0.25.0"                                                   : version0.25.md
```

Comment:
Indentation must be consistent: please use spaces here.

Reply:
Updated.